src/main/webapp/assets/modules/employer-dashboard/employerDashboardController.js

```diff
@@ -77,8 +77,9 @@ techlooper.controller('employerDashboardController', function ($scope, jsonValue
           if (index < 0) {
             return;
           }
-          $scope.dashboardInfo.challenges.splice(index, 1);
-          if (!$scope.filterChallenges().length) $scope.changeChallengeStatus();
+          $scope.dashboardInfo.notStartedChallenges.splice(index, 1);
+          //data.closedChallenges = $filter("progress")(data.challenges, "challenges", jsonValue.status.closed);
+          if (!$scope.dashboardInfo.notStartedChallenges.length) $scope.changeChallengesByStatus();;
```

Comment:
Line is too long.<br>Expected '{' and instead saw '$scope'.<br>Unnecessary semicolon.